data-plane/tests/src/test/java/dev/knative/eventing/kafka/broker/tests/DataPlaneTest.java

```diff
@@ -189,20 +191,12 @@ public void execute(final Vertx vertx, final VertxTestContext context) throws Ex
       )
       .build();
 
-    final var reconciler = ResourcesReconcilerImpl
-      .builder()
-      .watchIngress(requestHandler)
-      .watchEgress(consumerDeployer)
-      .build();
-
-    final var waitReconciler = new CountDownLatch(1);
+    new ContractPublisher(vertx.eventBus(), ResourcesReconcilerMessageHandler.ADDRESS)
+      .accept(DataPlaneContract.Contract.newBuilder().addResources(resource).build());
 
-    // reconcile ingress/egress
-    reconciler.reconcile(Collections.singletonList(resource))
-      .onFailure(context::failNow)
-      .onSuccess(v -> waitReconciler.countDown());
-
-    waitReconciler.await(TIMEOUT, TimeUnit.SECONDS);
+    //TODO(slinkydeveloper) for testing purpose, we need to implement a way to propagate results of reconciliation
+    // Or should we use awaitability or similar?
+    Thread.sleep(10000);
```

Comment:
For this one I'm thinking to add (later, in another pr) http://www.awaitility.org/ and wait on the number of deployed verticles, wdyt @pierDipi ?

Reply:
SGTM

Reply:
Can you open an issue?

Reply:
done https://github.com/knative-sandbox/eventing-kafka-broker/issues/351